submissions/TaniaBondarenko/frogger_game/app.js

```diff
@@ -2,62 +2,76 @@ const borders = {
     rightBorder: 400,
     leftBorder: 5,
     upBorder: 410,
-    downBorder: 70
+    downBorder: 5
 };
+const someGap = 10;
 
-const Enemy = function (x, y) {
-    this.sprite = 'images/enemy-bug.png';
-    this.x = x;
-    this.y = y;
-    this.speed = 75;
-
-    this.render = function () {
+class GameSprite {
+    constructor (x, y,sprite)  {
+        this.sprite = sprite;
+        this.x = x;
+        this.y = y;
+    }
+    render = function () {
         ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
     };
+}
```

Comment:
If you are using ES6 classes 'render' method must be defined in the same way as 'constructor':

```
render() {
    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
};
```

Reply:
So, If I use classes - all methods/function I put there. If functions - then I use Prototypes to define methods/functions. Right?

Reply:
yes, you are right!

Reply:
Okay. Thank you. I'll finish tomorrow and let you know.
Good night!)